app/assets/stylesheets/ama_layout/layout/base.scss

```diff
@@ -8,172 +23,255 @@ html{
 }
+
 .panel-box-error{
   background: $brand-red;
 }
-.image-callout{
-  border: $base-border;
+
+/* ==================== 3. Positioning ==================== */
+.left{
+  float: left !important;
 }
+
 .right{
-  float:right;
-}
-.left{
-  float:left;
+  float: right !important;
 }
+
 .center{
   margin-left: auto;
   margin-right: auto;
 }
+
 .block{
   display: block !important;
 }
+
 .inline{
   display: inline !important;
 }
+
 .inline-block{
   display: inline-block !important;
 }
-.left{
-  float: left !important;
-}
-.right{
-  float: right !important;
-}
+
 .text-left{
   text-align: left !important;
 }
+
 .text-center{
   text-align: center !important;
 }
+
 .text-right{
   text-align: right !important;
 }
-.mt0{
-  margin-top: 0 !important;
+
+/* ==================== 4. Margins ==================== */
+.m1{
+  margin: $base-margin*4 !important;
 }
+
 .mt1{
   margin-top: $base-margin*4 !important;
 }
-.mb0{
-  margin-bottom: 0 !important;
-}
-.mb1{
-  margin-bottom: $base-margin*4 !important;
-}
-.mr0{
-  margin-right:0px !important;
-}
+
 .mr1{
   margin-right: $base-margin*4 !important;
 }
-.ml0{
-  margin-left:0px !important;
+
+.mb1{
+  margin-bottom: $base-margin*4 !important;
 }
+
 .ml1{
   margin-left: $base-margin*4 !important;
 }
 
-.pl0 {
-  padding-left:0 !important;
+.m0{
+  margin: 0 !important;
 }
-.pr0 {
-  padding-right:0 !important;
+
+.mt0{
+  margin-top: 0 !important;
 }
-.pb0 {
-  padding-bottom:0 !important;
+
+.mr0{
+  margin-right: 0px !important;
 }
-.pt0 {
-  padding-top:0 !important;
+
+.mb0{
+  margin-bottom: 0 !important;
+}
+
+.ml0{
+  margin-left: 0px !important;
 }
+
+/* ==================== 5. Padding ==================== */
+.p1{
+  padding: $base-padding*4;
+}
+
 .pt1 {
   padding-top: $base-margin*4 !important;
 }
+
+.pr1 {
+  padding-right: $base-margin*4 !important;
+}
+
 .pb1 {
   padding-bottom: $base-margin*4 !important;
 }
 
-.bb0{
-  border-bottom:0px !important;
+.pl1 {
+  padding-left: $base-margin*4 !important;
 }
 
-.fullwidth{
-  width:100%;
+.p0{
+  padding: 0 !important;
 }
-.white-trans{
-  background-color: $white;
-  -ms-filter: "progid:DXImageTransform.Microsoft.Alpha(Opacity=90)";
-  background-color: rgba(255,255,255,0.6);
+
+.pt0 {
+  padding-top: 0 !important;
 }
 
-//bg colour
+.pr0 {
+  padding-right: 0 !important;
+}
+
+.pb0 {
+  padding-bottom :0 !important;
+}
+
+.pl0 {
+  padding-left: 0 !important;
+}
+
+.bb0{
+  border-bottom: 0px !important;
+}
 
+/* ==================== 6. Background & Text Colours ==================== */
 .whitebg{
-  background-color:$white;
+  background-color: $white;
 }
+
 .greybg{
-  background-color:$stone;
+  background-color: $stone;
 }
+
 .insurance-blue-bg{
-  background-color:$brand-blue-insurance;
+  background-color: $brand-blue-insurance;
+}
+
+.light-blue-bg{
+  background: $brand-blue-light;
 }
 
+.dark-blue-bg{
+  background: $brand-blue-dark;
+}
+
+.rewards-green-bg{
+  background-color: $brand-green;
+}
+
+.white-text{
```

Comment:
Should this be in the typography file? Makes sense here too since it's a base style so I guess it depends on what we want in the typography file.